cli/stores.hpp

```diff
@@ -468,7 +468,7 @@ void show<fiction::cell_layout_t>(std::ostream& os, const fiction::cell_layout_t
                                           lyt_ptr->get_layout_name())
                            << std::endl;
         }
-        else if constexpr (fiction::has_qca_technology_v<Lyt> && !fiction::is_offset_ucoord_v<Lyt>)
+        else if constexpr (fiction::has_qca_technology_v<Lyt> && !fiction::is_offset_ucoord_v<fiction::coordinate<Lyt>>)
```

Comment:
I think this was supposed to be 
```suggestion
        else if constexpr (fiction::has_qca_technology_v<Lyt> && !fiction::has_offset_ucoord_v<Lyt>)
```

Reply:
Yes, makes sense.